release-notes/EPM/I-DOCSIS/EPM_7.0.1_I-DOCSIS.md

```diff
@@ -0,0 +1,68 @@
+---
+uid: EPM_7.0.1_I-DOCSIS
+---
+
+# EPM 7.0.1 I-DOCSIS - preview
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be modified or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### Channel Utilization added for US/DS Service Group [ID_38849]
+
+When you open the ​Upstream or Downstream Service Group entities in the Topology app, the Channel Utilization average of the associated channels is now displayed in the data section.
+
```

Comment:
In the EPM topology chain, each level is a chain.
*When you open the Upstream or Downstream Service Group chains* in the topology app ...

Reply:
The chain is the whole topology, each level is a field. Should I specify _When you open the ​Upstream or Downstream Service Group entities under the DOCSIS Service Group Chain_?

Reply:
Indeed, it was a mistake on my side. Each level is a field.
It should be *When you open the Upstream or Downstream Service Group levels under the DOCIS Service Group chain*